.github/workflows/update-intellij-version.yml

```diff
@@ -0,0 +1,34 @@
+name: Update IntelliJ Version
+
+on:
+  schedule:
+    - cron: '0 1 * * 1'
+
+jobs:
+  bump-intellij-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.0.2
+      - name: Update default IntelliJ version
+        run: |
+          sbt "update-intellij / run --java-home /usr/local/openjdk-11"
+      - name: Prepare variables for PR
+        # language=sh
+        run: |
+          release_line=$(grep 'release = \"' core/driver/sources/src/main/resources/reference.conf)
+          intellij_version=$(awk -F'\"|\"' '{print $2}' <<< "$release_line")
+          echo "BRANCH_NAME=update_intellij_version_to_$intellij_version" >> $GITHUB_ENV
+          echo "PR_TITLE=Update IntelliJ version to $intellij_version" >> $GITHUB_ENV
+          scala_plugin_file_url="https://github.com/VirtusLab/ide-probe/blob/master/extensions/scala/tests/src/test/resources/scala.conf"
+          scala_plugin_versions_url="https://plugins.jetbrains.com/plugin/1347-scala/versions"
+          echo "PR_BODY=Updates the default IntelliJ Version to the newest release. \
+            Please, check currently used version of the scala plugin: $scala_plugin_file_url \
+            and verify if it is compatible with the new intelliJ IDEA version ($intellij_version) here: $scala_plugin_versions_url. \
+            If the version of the scala plugin used in $scala_plugin_file_url is NOT compatible - please update it in a separate commit." >> $GITHUB_ENV
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          branch: ${{ env.BRANCH_NAME }}
+          base: "master"
+          title: ${{ env.PR_TITLE }}
```

Comment:
Since you're **not** committing explicitly, I'm assuming that the action performs the commit, filling up the commit message from `PR_TITLE` right?

Reply:
No, the commit message by default is `[create-pull-request] automated change`. Let me change that by adding the `commit-message` action input in upcomming commit.